src/core/src/bootstrap/Constants.py

```diff
@@ -197,6 +197,7 @@ class RebootStatus(EnumBackport):
         REQUIRED = "Required"
         STARTED = "Started"
         COMPLETED = "Completed"
+        COMPLETED_WITH_WARNINGS = "CompletedWithWarnings"
```

Comment:
The ask is on patch operation status, not on reboot status. Hold off on additional iterations until discussion is cleared.